backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/EmailSenderService.cs

```diff
@@ -17,7 +19,29 @@ public EmailSenderService(IProducerService producerService)
     public void Send(EmailDto email)
     {
         var message = JsonConvert.SerializeObject(email);
-        
+
         _producerService.Send(message, "application/json");
     }
+
+    public EmailDto CreateEmailSubjectAndBody(UserDto currentUser, User userToInvite, Team team, string link)
+    {
+        return new EmailDto
+        {
+            Subject = $"{currentUser.UserName} wants to add you to the team {team.Name} on Easymeets",
+            Body = $"Hi {userToInvite.Name},\r\nUser {currentUser.UserName} would like to add to the team {team.Name}.\r\n" +
+                   $"Please follow the link. {link}\r\nIf this was a mistake please ignore this email.",
+            Recipient = userToInvite.Email
+        };
+    }
+
+    public EmailDto CreateEmailSubjectAndBody(UserDto currentUser, string email, Team team, string link)
+    {
+        return new EmailDto
+        {
+            Subject = $"{currentUser.UserName} wants to add you to the team {team.Name} on Easymeets",
```

Comment:
Subject and body are duplicated several times. It can be moved probably somewhere and then be reused